modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -35,21 +35,25 @@ def submit_form_2122 # rubocop:disable Metrics/MethodLength
           file_number = check_file_number_exists!
           claimant_information = validate_dependent_claimant!(poa_code:)
 
-          power_of_attorney = ClaimsApi::PowerOfAttorney.find_using_identifier_and_source(header_md5:,
-                                                                                          source_name:)
+          primary_identifier = {}
+          primary_identifier[:header_hash] = header_hash || primary_identifier[:header_md5] = header_md5
```

Comment:
Just something to consider, but I'm finding this line difficult to grok and wonder if it would be more understandable if it was split into separate lines/assignments.